apps/console/src/extensions/i18n/models/extensions.ts

```diff
@@ -1614,6 +1614,14 @@ export interface Extensions {
                         description: string;
                         message: string;
                     };
+                    successWaiting: {
```

Comment:
Check if we need to add them to the other repo

Reply:
Ack.